### ISSUE TITLE
only downloading 800 posts on twitter

### ISSUE THREAD

**Issue by Mas99** [AAndyProgram/SCrawler#155]
Hi,

Not sure if it is my setting but SC is only downloading the 1st 800 posts on twitter ? How do I change it ?

**Comment by AAndyProgram** [comment]
There is nothing to change. Twitter has always been limited to parsing only the top 1000 posts. Currently twitter has gone completely crazy. So the fact that you downloaded 800 posts is already good.

**Comment by sriwijaya572** [comment]
Hi @AAndyProgram is there any way to continue download past 1k media?
like there will be countdown every 500 download, depreciated extension "Twitter media downloader" use this method to bypass it that time. but eventually twitter api become paid and the extension got depreciated by them

**Comment by AAndyProgram** [comment]
@sriwijaya572 no
SCrawler uses [gallery-dl](https://github.com/mikf/gallery-dl) to download twitter. You should ask there.

----

**Join our Discord server**: https://discord.gg/uFNUXvFFmg

**Comment by sriwijaya572** [comment]
okay
thanks

**Comment by Sambogreen9** [comment]
Fagg

**Comment by AAndyProgram** [comment]
@krystian2160, @sriwijaya572, guys, did you read this issue carefully? As @mikf [said](https://github.com/mikf/gallery-dl/issues/5860#issuecomment-2234587656):
> `--sleep` and `--sleep-request` alone are **not enough to keep Twitter from suspending your account** it seems, but I also can't tell you what is.

**Comment by krystian2160** [comment]
So you mean it doesnt work cuz you did not figure out how to do it because sleep is not good enough alone so it works?

**Comment by AAndyProgram** [comment]
> So you mean it doesnt work cuz you did not figure out how to do it because sleep is not good enough alone so it works?

I didn't add it because it might lead to account ban

**Comment by krystian2160** [comment]
Might
But with good sleep time... JDownloader managaes somehow full crawl with no ban.
But I had to do in settings 20.000 ms for every request, so there is no short-time ban (throttle or something).

**Comment by AAndyProgram** [comment]
I'm willing to consider adding this feature if someone provides me with a Twitter account to tests. Sorry, but I won't debug features that could get my real account banned. 😉

**Comment by sriwijaya572** [comment]
Hey Andy its me again
I noticed several people began to complain about limited download from twitter and instagram post

Its somewhere 800 to 1000

And you said you need decoy account for debug it
I have 2 here, did you still need it?

**Comment by krystian2160** [comment]
Yes! That would be so great if Andy agrees on that! Nice one bro @sriwijaya572

**Comment by AAndyProgram** [comment]
> And you said you need decoy account for debug it
> I have 2 here, did you still need it?

@sriwijaya572 DM me on Discord

**Comment by krystian2160** [comment]
Any progress on that? ^^ Im very interested and curious! @sriwijaya572 @AAndyProgram

**Comment by krystian2160** [comment]
I see the issue is closed so its done and works? ^^

**Comment by AAndyProgram** [comment]
@krystian2160 https://github.com/AAndyProgram/SCrawler/issues/155#issuecomment-2403290888

**Comment by krystian2160** [comment]
@AAndyProgram didnt @sriwijaya572 provide X accounts?
if not please tell me I can provide.

**Comment by AAndyProgram** [comment]
Will be added in the next version!

----

> [@AAndyProgram](https://github.com/AAndyProgram) didnt [@sriwijaya572](https://github.com/sriwijaya572) provide X accounts? if not please tell me I can provide.

Provided!

----

**Join our Discord server**: https://discord.gg/uFNUXvFFmg

**Comment by AAndyProgram** [comment]
**Added**. Please update to the [latest release](https://github.com/AAndyProgram/SCrawler/releases/latest).
